### PR TITLE
Create collatz conjecture function

### PR DESCRIPTION
**Pull request by lumodon**

### CODE REVIEW

File: test/collatz_test.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+describe('collatz()', function(){
+
+  it('should be a function', function(){
+    expect(collatz).to.be.a('function')
+  })
+
+  it('returns \'1\' no matter what ordinal natural number is passed to it.', function(){
+    expect(collatz(59)).to.equal(1)
+    // expect(collatz(199)).to.equal(1)
```
Comment (shakalee14) [inline test/collatz_test.js]: please remove commented code on line 12

File: src/collatz.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export default function makeChange(collatzerand) {
+  if(collatzerand <= 0 || collatzerand % 1 !== 0) return 'Please use only ordinal natural numbers. (1,2,3...)'
```
Comment (shakalee14) [inline src/collatz.js:2]: nice message

Comment (shakalee14) [reply src/collatz.js:2]: just a question -- you would get this message for infinity too right?

**Review by shakalee14 — COMMENTED** [review]
lgtm!

File: src/collatz.js (hunk 2 above)
Comment (lumodon) [reply src/collatz.js:2]: Yes indeed. Just as expected from http://www.wolframalpha.com/input/?i=infinity+modulo+1

**Pull request merged**